gwpy/io/tests/test_kerberos.py

```diff
@@ -35,6 +35,7 @@
 KVNO Principal
 ---- -------------------------------
    1 albert.einstein@LIGO.ORG
+   2 ronald.drever@LIGO.ORG
```

Comment:
@duncanmmacleod, is this intentionally duplicated?

Reply:
@alurban, yes, to check that the `ronald.drever` principal only gets returned once.